@Goodnews-code/gitcollab-ui-task.md

```diff
@@ -6,12 +6,9 @@ Hi, I'm @Goodnews-code. I'm currently learning to collaborate with Git and GitHu
 
 Unordered list used for listing markdown examples below:
 - Link: [This link text will take you to Collabo Community's Github Page when you click on it!](https://github.com/collabo-community)
-- Inline Code Block: `This code block text will show up inline`. Do you notice the difference?
+
 - Bold text: **This text will show up as a Bold Text**
-- Code Block:
-````
-This code block text will show up as a separate block
-````
```

Comment:
@Goodnews-code Your solution here is incorrect. you were not directed to remove the Code block list item.